filters/conversationfilter.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+from telegram.ext import BaseFilter
+
+from conversationrequest import ConversationType
+from conversations import Conversations
+
+
+class ConversationFilter(BaseFilter):
+    def filter(self, message):
+        """Returns True if the sender of the message is in a conversation"""
+        conversations = Conversations()
+        return conversations.get_conversation(message.from_user.id) is not None
```

Comment:
Implementing it like there will make it not possible to run the filter on updates which don't have messages, but user_ids, like poll_answers. I am not going to say it's likely, but we can avoid this unnecessary restriction if we change it to an update filter.


```suggestion
class ConversationFilter(BaseFilter):
    
    update_filter = True
    
    def filter(self, update):
        """Returns True if the sender of the update is in a conversation"""
        conversations = Conversations()
        # This check is needed in case someone adds the bot to a channel or sends a Poll
        if update.effective_user:
            return conversations.get_conversation(update.effective_user.id) is not None
        return None
```

Reply:
what is update_filter doing, where is it used?

Reply:
Used in filtering incoming updates from Telegram on PTB handler level

Reply:
Following the KISS and the YAGNI principles, I don't want to implement this now. We don't plan on using polls at the moment, so working on messages is enough atm. And the change can be easily done in the future, but I appreciate your objection.